### PR TITLE
Times, alarms ++

### PR DESCRIPTION
**Pull request by 10cents**

### CODE REVIEW

File: commons/src/main/res/values-da/strings.xml [hunk 1]
```diff
@@ -233,22 +233,22 @@
...
+        <item quantity="one">%d time</item>
+        <item quantity="other">%d timer</item>
     </plurals>
 
     <!-- For example: The alarm will go off in 6 hours and 5 minutes -->
-    <string name="alarm_goes_off_in_hours_minutes">The alarm will go off in %1$d hours and %2$d minutes</string>
-    <string name="alarm_goes_off_in_hours">The alarm will go off in %d hours</string>
-    <string name="alarm_goes_off_in_minutes">The alarm will go off in %d minutes</string>
+    <string name="alarm_goes_off_in_hours_minutes">Alarmen starter om %1$d timer og %2$d minutter</string>
+    <string name="alarm_goes_off_in_hours">Alarmen starter om %d Timer</string>
```
Comment (tibbi) [inline commons/src/main/res/values-da/strings.xml]: is capital "T" in "Timer" proper here? it is small on line 249

File: commons/src/main/res/values-da/strings.xml [hunk 2]
```diff
@@ -265,11 +265,11 @@
     <string name="password_protect_whole_app">Beskyt hele appen med en adgangskode</string>
     <string name="keep_last_modified">Kopi/flyt/omdøb skal ikke påvirke ændringstidspunktet</string>
     <string name="show_info_bubble">Vis en info-boble når jeg scroller med rullepanelet</string>
-    <string name="prevent_phone_from_sleeping">Forhindr telefonen i at falde i søvn når appen er i forgrunden</string>
+    <string name="prevent_phone_from_sleeping">Prevent phone from sleeping while the app is in foreground</string>
```
Comment (tibbi) [inline commons/src/main/res/values-da/strings.xml]: guess you didn want to override this one?

File: commons/src/main/res/values-da/strings.xml [hunk 3]
```diff
@@ -354,11 +354,11 @@
...
         <![CDATA[
-            Hej<br><br>
-            - synes lige du skulle vide at der er kommet en ny app:<br><br>
+            Hey,<br><br>
+            just letting you know that a new app has been released recently:<br><br>
             <a href="%1$s">%2$s</a><br><br>
-            Du kan downloade den med et klik på titlen.<br><br>
-            Tak
+            You can download it by pressing the title.<br><br>
+            Thanks
```
Comment (tibbi) [inline commons/src/main/res/values-da/strings.xml]: also not overriding these

File: commons/src/main/res/values-da/strings.xml (hunk 1 above)
Comment (10cents) [reply commons/src/main/res/values-da/strings.xml]: You're right, thanks

File: commons/src/main/res/values-da/strings.xml (hunk 2 above)
Comment (10cents) [reply commons/src/main/res/values-da/strings.xml]: I do not understand, why not?

File: commons/src/main/res/values-da/strings.xml (hunk 3 above)
Comment (10cents) [reply commons/src/main/res/values-da/strings.xml]: Not?

File: commons/src/main/res/values-da/strings.xml (hunk 2 above)
Comment (10cents) [reply commons/src/main/res/values-da/strings.xml]: Ha, sorry, no.

File: commons/src/main/res/values-da/strings.xml (hunk 3 above)
Comment (10cents) [reply commons/src/main/res/values-da/strings.xml]: no :-)
I must have made suggestions to the file too early after the previous suggestions.

**Pull request merged**